cogs/score.py

```diff
@@ -42,146 +43,217 @@ async def on_ready(self):
         self.DOWNMARTLET = discord.utils.get(
             self.guild.emojis, name=self.bot.config.downvote_emoji)
 
-    def _get_score(self, emoji):
-        if emoji.id == self.UPMARTLET.id:
-            return 1
-
-        elif emoji.id == self.DOWNMARTLET.id:
-            return -1
-
-        return 0
-
-    async def _update_score_if_needed(self, payload, negated=False):
-        # Check for Martlet emoji + upmartletting yourself
+    async def _get_name_from_id(self, user_id):
+        try:
+            user = self.bot.get_user(user_id)
+            if user is None:
+                raise AttributeError
+            name = str(user)
+        except AttributeError:
```

Comment:
does this occur somewhere other than the if statement above? if so from what?

Reply:
I don't think so, changed it to if statement in 8a67002